### PR TITLE
Removing single character variables conflicting with debug commands.

### LINKED ISSUE #6: Add support for fields in Identify for Youtube.
See #5 
Request for adding support for the Identify fields.

### PR DESCRIPTION
**Pull request by ankenyr**
Fixes https://github.com/ankenyr/jellyfin-youtube-metadata-plugin/issues/6

### CODE REVIEW

File: migrate.py [hunk 1]
```diff
@@ -70,24 +70,24 @@ def migrate(plex_url: str, plex_token: str, jellyfin_url: str,
...
         else:
             if debug:
-                print(f"{bcolors.OKBLUE}{w['title']}{bcolors.ENDC}")
+                print(f"{bcolors.OKBLUE}{watched['title']}{bcolors.ENDC}")
 
     print(f"{bcolors.OKGREEN}Succesfully migrated {len(plex_watched)} items{bcolors.ENDC}")
 
 
-def _search(data: dict, item: dict) -> List:
+def _search(lib_data: dict, item: dict) -> List:
```
Comment (wilmardo) [inline migrate.py:90]: Could you update `data` to `lib_data` in the docsting too?

**Comment by wilmardo** [comment]
Massive thanks for the contribution!

I have just one nitpick for the docstring, otherwise it looks fine to me :)

File: migrate.py (hunk 1 above)
Comment (ankenyr) [reply migrate.py:90]: done! Sorry that was left for over a month. Life got in the way for a while.

**Comment by ankenyr** [comment]
This should be done, sorry again for letting this linger. Also thank you so much for making this as it really helped me migrate away from Plex. I owe you a drink.

**Pull request merged**